app/scheduling/templates/SchedulingCtrl.html

```diff
@@ -72,7 +72,10 @@
 							<small class="text-muted">{{ view.state.courses.list[view.state.sectionGroups.list[sectionGroupId].courseId].title }}</small>
 							<br />
 							<small class="text-muted">
-								Units: {{ view.state.courses.list[view.state.sectionGroups.list[sectionGroupId].courseId].unitsLow }}
+								Units: {{ view.state.courses.list[view.state.sectionGroups.list[sectionGroupId].courseId].unitsHigh
+													? view.state.sectionGroups.list[sectionGroupId].unitsVariable || (view.state.courses.list[view.state.sectionGroups.list[sectionGroupId].courseId].unitsLow + " - " + view.state.courses.list[view.state.sectionGroups.list[sectionGroupId].courseId].unitsHigh)
```

Comment:
We mess up this logic a lot, so I would instead add a transient helper on the entity in the backend so you just see a displayUnits string property which will be the correct thing to show